<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PubDlg</class>
 <widget class="QDialog" name="PubDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labFront">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>421</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labMyRole">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>70</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>我的角色：发布者</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>119</y>
     <width>501</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labPncp">
      <property name="text">
       <string>负责人报名情况</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labTrans">
      <property name="text">
       <string>译文提交情况</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPlainTextEdit" name="plainTextEditTrans">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="plainText">
       <string>暂未完成</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>399</y>
     <width>91</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="confirmBtn">
      <property name="text">
       <string>确认完成</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="payBtn">
      <property name="text">
       <string>支付酬金</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="depositBtn">
      <property name="text">
       <string>充值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="quitBtn">
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitBtn</sender>
   <signal>clicked()</signal>
   <receiver>PubDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>536</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
